ansible/uod-docker.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Playbook for managing a docker instance in the
+# openstack cloud.
+
+- hosts: docker-hosts
+  vars:
+    ansible_user: centos
+  roles:
+  - role: storage-volume-initialise
+    storage_volume_initialise_device: /dev/vdb
+    storage_volume_initialise_mount: /var/lib/docker
+  - role: basedeps
+  - role: system-monitor-agent
+  - role: versioncontrol-utils
+  - role: docker
+    docker_groupmembers: [centos]
+  - role: sudoers
+  - role: local-accounts
```

Comment:
I'll try this now. The failure I've been getting is:

```
TASK [omero-server : omero | enable server systemd] ****************************
fatal: [training-server]: FAILED! => {"changed": false, "failed": true, "msg": "Job for omero.service failed. See 'systemctl status omero.service' and 'journalctl -xn' for details.\n"}
```

```
~/opt/management_tools/infrastructure/ansible $ vagrant ssh training-server
Last login: Thu Jul 14 23:44:12 2016 from 10.0.2.2
[vagrant@localhost ~]$ sudo systemctl status omero.service
omero.service - OMERO.server
   Loaded: loaded (/etc/systemd/system/omero.service; enabled)
   Active: failed (Result: timeout) since Thu 2016-07-14 23:49:12 EDT; 25s ago
  Process: 19628 ExecStart=/home/omero/OMERO.server/bin/omero admin start (code=killed, signal=TERM)
```

```
Caused by: org.postgresql.util.PSQLException: FATAL: password authentication failed for user "omero"
```